README.md

```diff
@@ -218,9 +218,14 @@ Find the ACORD year/month at the bottom left corner of each ACORD page. Often th
 
 - [CMS 1500 (08/05)](./health_insurance_claims/cms_1500/cms_1500_08_05.json)
 
-### HR
+### hr
+
+- [Resume](./hr/resume/resume.json)
+
+### Drivers License
```

Comment:
We should be grammatically correct here when it's a label (and below) -- Driver's License